BioCro/DARPA/upload_params.Rmd

```diff
@@ -143,3 +143,35 @@ write.csv(sla, file = "upload_params_data/sla.csv", row.names = FALSE)
 ```
 
 Upload values to Welsch BETYdb using the same method as for physiological parameters. 
+
+## Physiological parameter cuticular conductance
+
+Has most of the same columns as the first set of parameters: `notes`, `local_datetime`, `treatment`, with `cuticular_cond` for the values. 
+
+```{r}
+upload_cc <- all_parameters %>% 
+  mutate(notes = paste("row", row_number(), "in https://docs.google.com/spreadsheets/d/1doQI0T7vav7LmNdMEoZJDx9iY_Imqa229CLRbRvurFc/edit#gid=967233489")) %>% 
```

Comment:
I don't have access to this sheet - can you make it public? Also, the link is for editing the sheet `.../edit#gid=967233489` so it should probably change.

Reply:
I can't make that data public, but I might be able to give you access to it. @dlebauer?